bbmod/bbmod.vcxproj

```diff
@@ -21,7 +21,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{A822A2F2-38F1-43B1-ACCD-3DBBD3F66C1B}</ProjectGuid>
     <RootNamespace>bbmod</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>

```

Comment:
I'd prefer to leave the target platform version the same for now, I'm not sure what the implications are of this changing.